### PR TITLE
style(btn): btn gradient updated

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**
Gradient:
Btn background gradient => radial-gradient(circle, rgb(27 0 255) 13%, rgb(0 30 247 / 86%) 21%, rgb(107 90 234) 91%);
Btn on-hover gradient =>linear-gradient(to right, #12d3ff 0%, #921cff 103%);

Kindly let me know if we need to make changes in gradient shades.

### CODE REVIEW

**Comment by Shaharyar-saleem** [comment]
if chainId != 56 then error screen is displaying

**Comment by Shaharyar-saleem** [comment]
error msg needs to change as well

File: assets/js/connectWallet.js [hunk 1]
```diff
@@ -13,106 +13,112 @@ let selectedAccount;
...
+    // Because rendering account does its own RPC commucation
+    // with Ethereum node, we do not want to display any results
+    // until data for all accounts is loaded
+    await Promise.all(rowResolvers);
+
+    // Display fully loaded UI for wallet data
+    document.querySelector("#prepare").style.display = "none";
+    document.querySelector("#connected").style.display = "block";
+    // Display Network Error
+    if (chainId != (56 || 97)) {
```
Comment (bobachef) [inline assets/js/connectWallet.js:117]: this is wrong.. 

And this proves to me that you didn't test your own code before telling me

**Review by bobachef — COMMENTED** [review]
Please test your code before you tell me to

**Review by bobachef — CHANGES_REQUESTED** [review]
please test your code and fix bugs before you ask me to take a look

File: assets/js/connectWallet.js (hunk 1 above)
Comment (bobachef) [reply assets/js/connectWallet.js:117]: this is simple javascript...

(56 || 97) evaluates to 56 

so you're not checking for 97

**Comment by Shaharyar-saleem** [comment]
code has been tested.

**Review by bobachef — APPROVED** [review]
lgtm. Just open a new PR with the reload fix. Thank you.

**Pull request merged**